web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FedoraContentController.java

```diff
@@ -154,4 +164,20 @@ public ResponseEntity<Object> handleArgumentTypeMismatch(RuntimeException ex) {
         log.debug("Argument type mismatch", ex);
         return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
     }
+
+    public void setFedoraContentService(FedoraContentService fedoraContentService) {
```

Comment:
were these added for the tests?

Reply:
yes, since it's using a standalone server with a specific controller passed in, rather than being injected by spring